### PR TITLE
Kill Firecracker forcefully if StopVM is hitting its context's timeout

### PR DESCRIPTION
**Pull request by kzys**
This commit doesn't fix the issue, but reproduce that in
TestStopVM_Isolated.

Signed-off-by: Kazuyoshi Kato <katokazu@amazon.com>

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by kzys** [comment]
Finally worked! I will separate the last three commits from this PR next week.

File: runtime/service_integ_test.go [hunk 1]
```diff
@@ -2237,8 +2256,9 @@ func TestPauseResume_Isolated(t *testing.T) {
 	require.NoError(t, err, "failed to create ttrpc client")
 
 	subtests := []struct {
-		name  string
-		state func(ctx context.Context, resp *proto.CreateVMResponse)
+		name             string
+		state            func(ctx context.Context, resp *proto.CreateVMResponse)
+		forcefulShutdown bool
```
Comment (austinvazquez) [inline runtime/service_integ_test.go:2261]: Was this used?

Comment (kzys) [reply runtime/service_integ_test.go:2261]: It is not. Let me undo this change.

**Review by austinvazquez — APPROVED** [review]
Awesome!

**Pull request merged**